### ISSUE TITLE
Created a new placeholder, when I look at the "Edit" on the LHS it tells me the number is taken in red

### ISSUE THREAD

**Issue by allanchaplin** [nittygritty-uk/Atvero-Issues#82]
Whilst I suppose it is technically correct...

![screenshot 2018-12-10 at 12 17 28](https://user-images.githubusercontent.com/53964/49732253-eea58c00-fc75-11e8-8255-c5c97745599f.png)

**Comment by LBoyle** [comment]
Yep, still haven't spent time on validating numbers since going BS1192 compatible

**Comment by LBoyle** [comment]
This is fixed in PR #142
